erdkotlin/src/main/java/com/elrond/erdkotlin/data/vm/responses/QueryContractResponse.kt

```diff
@@ -0,0 +1,25 @@
+package com.elrond.erdkotlin.data.vm.responses
+
+import com.elrond.erdkotlin.domain.vm.SmartContractOutput
+import com.google.gson.internal.LinkedTreeMap
+import java.math.BigInteger
+
+internal data class QueryContractResponse(
+    val data: Data
+) {
+
+    data class Data(
+        val returnData: List<String>?, // ex: ["Aw=="]
+        val returnCode: String, // ex: "ok"
+        val returnMessage: String?,
+        val gasRemaining: BigInteger,
+        val gasRefund: BigInteger,
+        val outputAccounts: LinkedTreeMap<String, SmartContractOutput.OutputAccount>,
+
+        // documentation is not clear about this
```

Comment:
Do you think you can point to this file instead, for the moment?

https://github.com/ElrondNetwork/elrond-go/blob/master/core/vmcommon/output.go

Reply:
Done 👍